src/components/scenes/SwapCreateScene.tsx

```diff
@@ -531,7 +533,9 @@ export const SwapCreateScene = (props: Props) => {
             keyboardVisible={false}
             onAmountChanged={handleFromAmountChange}
             onNext={handleNext}
-            onSelectWallet={handleFromSelectWallet}
+            onSelectWallet={() => {
+              handleFromSelectWallet().catch(() => {})
+            }}
```

Comment:
I feel like this is the wrong solution. In `SwapInput.tsx`, we should have `onSelectWallet: () => Promise<void>`, and then if we use `EdgeTouchableOpacity` for `WalletPlaceHolder`, we get the correct promise handling.